server.js

```diff
@@ -2149,6 +2149,50 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Code Climate issues integration
+camp.route(/^\/codeclimate\/issues\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[1];  // eg, `github/me-and/mdf`.
+  var format = match[2];
+  var options = 'https://codeclimate.com/' + userRepo + '/badges/issue_count.svg'
+  var badgeData = getBadgeData('issues', data);
+  request(options, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      var count = buffer.match(/>([0-9]+) issues?/)[1];
+      if (!count) {
+        badgeData.text[1] = 'malformed';
+        sendBadge(format, badgeData);
+        return;
+      }
+      if (count == 1) {
+        badgeData.text[1] = '1 issue';
+      } else {
+        badgeData.text[1] = count + ' issues';
```

Comment:
Could you remove the `issue` part, please? I'd like the badge to just have a number on the right-hand side. Something like that:

![](https://img.shields.io/badge/issues-2-green.svg)